src/components/App/App.jsx

```diff
@@ -27,18 +33,94 @@ function App() {
   const [articlesObserv, setObservArticles] = useState(3);
   const [totalArticles, setTotalArticles] = useState(0);
   const [isSerchingStarted, setSerchingStart] = useState(false);
+  const [userArticles, setUserArticles] = useState(null);
+  const [keyword, setKeyword] = useState("");
+
+  const [currentUser, setCurrentUser] = useState(null);
+  const navigate = useNavigate();
+
+  //Authorization
+  useEffect(() => {
+    const token = localStorage.getItem("jwt");
+    if (token) {
+      getCurrentUser()
+        .then((res) => {
+          setCurrentUser(res);
+
+          setLoggedIn(true);
+        })
+        .catch(console.error);
+    }
+  }, [isLoggedIn]);
+
+  //Articles
+  useEffect(() => {
+    const token = localStorage.getItem("jwt");
+    if (token) {
+      getArticle()
+        .then((res)=> {
+          setUserArticles(res)
+        })
+        .catch(console.error);
+    }
+  }, [userArticles]);
```

Comment:
* There are contant server requests every second and the app freezes about of them https://snipboard.io/phZomx.jpg 


Please, delete `userArticles` from the array: it causes this constant fething

`  }, []);
`